src/utils/ConvertToEmbeddableUrl.tsx

```diff
@@ -8,28 +15,55 @@ export interface ConvertToEmbeddableUrlOptions {
 // @debt Replace the naive logic with use of new URL(url) which provides standard parsing before doing checks
 export const convertToEmbeddableUrl: (
   options: ConvertToEmbeddableUrlOptions
-) => string = ({ url, autoPlay = DEFAULT_VENUE_AUTOPLAY }) => {
-  if (url?.includes("youtube")) {
-    url = url?.replace("watch?v=", "embed/");
-  } else if (
-    url?.includes("vimeo") &&
-    !url?.includes("player") &&
-    // NOTE: If you have a scheduled live event, it gives you a different embed code
-    !url?.includes("vimeo.com/event")
+) => string = ({ url, autoPlay }) => {
```

Comment:
Can use destructuring to rename it internally `url: urlString` so it is made more clear later on:

```ts
  const urlObject = new URL(urlString);
```

or just

```ts
  const url = new URL(urlString);
```

Reply:
fixed ea1c3b0